Libraries/Image/RCTImageUtils.m

```diff
@@ -110,6 +111,26 @@ CGRect RCTTargetRect(CGSize sourceSize, CGSize destSize,
           RCTCeilSize(sourceSize, destScale)
         };
       }
+
+    case RCTResizeModeCenter:
+
+      // Make sure the image is not clipped by the target.
+      if (sourceSize.height > destSize.height) {
+        sourceSize.width = destSize.width = destSize.width;
```

Comment:
@javache Are you talking about `destSize.width = destSize.width` (which is used elsewhere so I just copied it)? Or do you mean I should be using the aspect ratios instead?